src/css/variables.styl

```diff
@@ -3,9 +3,6 @@
 css-prefix = 'tui-full-calendar-'
 // Dayname
 dayname = css-prefix + 'dayname-'
-// Taskview
-milestone = css-prefix + 'milestone-'
-taskview = css-prefix + 'task-'
 // Allday
 allday = css-prefix + 'allday-'
```

Comment:
It seems that the all day variable is also not used.
I think I can erase it, what do you think?